timebot/timebot.go

```diff
@@ -3,25 +3,62 @@
 package timebot
 
 import (
-	"strings"
+	"fmt"
+	s "strings"
 	"time"
 )
 
-// ParseTime takes a string and returns time.Time in time.UTC
-//
-// FIXME: Currently, it doesn't validate PST/PDT
+func CheckDaylightSavingZone(text string) (daylightSavingZone bool) {
+
+	tzDb := map[string]string{
+		"PST": "America/Los_Angeles",
+		"PDT": "America/Los_Angeles",
+	}
+
+	var tzText string = ""
```

Comment:
일단 lint error 대로 고쳐주시거나 아래로 고쳐주시는게 좋을 듯합니다.

```suggestion
	tzText := ""
```

Reply:
피곤하실텐데~
새해 복 많이 받으십시오. :-)